submissions/logistics_belts/logistics-belts.md

```diff
@@ -0,0 +1,27 @@
+
+## Future Work and Limitations
+
+### Multiple Destinations
+
+The problem of sending one item to multiple destinations is non-trivial, and it's something I haven't been able to personally solve. Because filter splitters pull all of an item off, sending that item past a filter splitter *some of the time only* requires complicated splitter arrangements and several circuit conditions originating from different destinations. I've given the problem a good deal of thought, and while I've come up with several prototypes they've mostly been prone to behaviors like jamming or sending too many of an item to the wrong destination, potentially causing an overflow if left unattended. With the Factorio community being what it is, I wouldn't be surprised if this is solved within a week or two, and in several different ways; maybe we'll even hear about *your* solution in a future Alt-F4. For now, though, I leave it as a challenge to the community. Be sure to let us know what you come up with!
+
+## A Note on Sushi Belts
+
+Players who have been around the game a while may have already encountered a similar concept: sushi belts. Like logistics belts, sushi belts place more than two resources on a belt, but they generally do so as a loop designed for complicated crafting. A circuit condition at the entrance or around the entire belt helps to control the amount of every resource present, even leading to such complicated examples as the "bloodbus" - a circulatory system for your factory (https://imgur.com/gallery/Q4oR0#Jiwed0Q). While the concept of a logistics belt is similar, it has a few important differences, and the key one is related to their function: while sushi belts go *around*, and tend to service multiple assemblers in no particular order, logistics belts go *to*, and tend to direct resources from one location to another. In fact, a logistics belt can be used to feed a sushi belt in place of an ordinary bus - when resources are needed, they can be sent, with carefully limited rate control preventing over-population of the belt loop. Other concepts like clocked belts or inserter limiting can be applied here - and can be combined with logistics belts to achieve monumental feats of engineering. Logistics belts themselves are just one part of the growing player's toolkit.
```

Comment:
Here could we embed a picture of the blood bus factory or just a regular sushi belt for anyone that hasn't seen them before?